.github/workflows/deploy-frontends-to-staging.yml

```diff
@@ -196,7 +199,7 @@ jobs:
       - run: |
           cd calibrate/
 
-          export REACT_APP_BASE_URL=${{ secrets.REACT_APP_BASE_URL }}
+          export REACT_APP_BASE_URL=${{ secrets.REACT_APP_BASE_URL_STAGE }}
```

Comment:
@NoahNsimbe, kindly update the repo secrets with `REACT_APP_BASE_URL_STAGE` accordingly

Reply:
@123MwanjeMike I think these were already configured by Daniel.